### DISCUSSION TITLE
Kong Konnect not showing Runtime EKS in Konnect Portal

### DISCUSSION THREAD

**Discussion by doris-dev** [Kong/kong#8803] [Help] [answered]
I'm trying to add a new Runtime to my EKS.
I already made all the correct settings:
https://docs.konghq.com/konnect/runtime-manager/gateway-runtime-kubernetes/

The Service/Pod is running normally, and there are no errors in the logs.
I can even access it through the generated LoadBalancer.
However, this Runtime client does not appear in the Konnect portal.
Is there any instability in the service or limit of Rutimes?
Because until 2 days ago I was able to add Runtimes without problems.

**Comment by mayocream** [comment, marked as answer]
Hi @doris-dev, this is the place to discuss/ask about Kong OSS. Could you please contact Konnect support to investigate this problem? Thanks!
